timemachine/ff/handlers/nonbonded.py

```diff
@@ -465,6 +471,12 @@ def static_parameterize(mol):
         """
         return oe_assign_charges(mol, "AM1BCCELF10")
 
+    def get_smirks(self, **kwargs):
+        return self.smirks
+
+    def get_params(self, **kwargs):
+        return self.params
```

Comment:
If `smirks` and `params` are intended to be private, should they be prefixed with `_`? (Also, `get_smirks` and `get_params` seem unused)

Reply:
removed these after refactoring.